### DISCUSSION TITLE
DVD complete iso image with all desktop distributions

### DISCUSSION THREAD

**Discussion by robodonster67** [SpiralLinux/SpiralLinux-project#355] [Q&A] [answered]
Hi, as my title states can all desktops be included on the iso image?

Thanks in advance 

Don

**Comment by geckolinux** [comment]
Hi there, no, that would make a mess. Additionally, some features of some desktops depend on a specific login manager.

**Comment by robodonster67** [comment]
DEbian 12 includes most of the desktop configurations..

**Reply by geckolinux** [reply]
Debian offers different desktop options, but it doesn't install them all simultaneously.

**Comment by marwenius** [comment, marked as answer]
If you want to install other desktop environments, you can run [tasksel](https://wiki.debian.org/tasksel) at your own risk.

**Comment by fweller** [comment]
Including all desktop environments in one single ISO file does not exist in the Debian world.  The only two distributions I am aware of that support installing any desktop environment from a single installer are OpenSUSE and NixOS.  OpenSUSE uses the YaST installer which to my understanding has not been ported to any other distribution, but it would be quite interesting if someone were to undertake this task.
As others have pointed out, you can switch your desktop environment after installation by running:
sudo apt install taskset
sudo taskset install <select one of these only: task-gnome-desktop | task-xfce-desktop | task-kde-desktop | task-lxde-desktop| task-cinnamon-desktop | task-mate-desktop | task-lxqt-desktop>
